src/test/collections/data/mps_verbdata.json

```diff
@@ -0,0 +1,58 @@
+[
+  {
+    "_TEST_CASE_6597": "MPS: Rest API-  http tampering- using PUT method for mps api's instead of POST",
+    "url": "http://{{host}}/api/v1/authorize",
```

Comment:
`http://` should be using the `{{procotol}}` env variable

Reply:
oh I missed this earlier. Let me use update that.

Reply:
Renamed files as well